src/parser/jobs/blm/modules/Sharpcast.js

```diff
@@ -0,0 +1,134 @@
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import React from 'react'
+import {Item} from 'parser/core/modules/Timeline'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Trans, Plural} from '@lingui/react'
+import {StatusLink} from 'components/ui/DbLink'
+
+const SHARPCAST_DURATION_MILLIS = STATUSES.SHARPCAST.duration * 1000
+
+const SHARPCAST_CONSUMER_IDS = [
+	ACTIONS.FIRE_I.id,
+	ACTIONS.THUNDER_III.id,
+	ACTIONS.THUNDER_IV.id,
+	ACTIONS.SCATHE.id,
+]
+
+export default class Sharpcast extends Module {
+	static handle = 'sharpcast'
+	static dependencies = [
+		'timeline',
+		'procs', // eslint-disable-line @xivanalysis/no-unused-dependencies
```

Comment:
Any reason you need procs? Doesn't look like you're using it, and there's no normalisers running.
If you're relying on execution order for something outside normalisers, I'd rather it be a side effect of a public api access.

Reply:
I did that to make sure the proc timeline group had already been set up before this module tries adding anything to it. I'm not 100% it's actually needed, I just put it there to be safe.

Reply:
Is an API now